test/testdata/deployednettemplates/recipes/network-partition/Makefile

```diff
@@ -0,0 +1,12 @@
+PARAMS=-w 100 -R 8 -N 20 -n 100 -H 10 --node-template node.json --relay-template relay.json --non-participating-node-template nonPartNode.json
```

Comment:
Do we really need such a big network to conduct a partitioning test ?
I don't really see how the large network would help us getting the answer "does the network recover after partitioning? and after how long ?"

Reply:
We can adjust the network to be smaller.  The github tickets ask for all 3 scenarios.

Reply:
That's true. It's that I just don't want you to get delayed by "deployment issues" that aren't related to the actual test you're trying to conduct.. Having the feedback that it's broken on small network is more valuable than the fact the it's broken on a large network ( and would probably be easier to evaluate )

Reply:
OK, we will start testing tomorrow.